tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -112,10 +115,21 @@ public BitmapProvider getBitmapProvider() {
     return mBitmapProvider;
   }
 
+  public BitmapRecycler getBitmapCleanup() {
+    if( mBitmapRecycler == null ){
+      mBitmapRecycler = new BitmapRecyclerDefault();
+    }
+    return mBitmapRecycler;
+  }
+
   public void setBitmapProvider( BitmapProvider bitmapProvider ) {
     mBitmapProvider = bitmapProvider;
   }
 
+  public void setBitmapRecycler(BitmapRecycler bitmapRecycler){
```

Comment:
whitespace
```
public void setBitmapRecycler( BitmapRecycler bitmapRecycler ){
```
i'd probably just format the whole file anytime you find a place where whitespace is off